### DISCUSSION TITLE
Fine-tune PP-OCRv4 DET Server Model

### DISCUSSION THREAD

**Discussion by lsch0lz** [PaddlePaddle/PaddleOCR#12304] [Q&A]
Hi!
I want to fine-tune the server detection model. Am I right assuming, that this is simply only the teacher model, without any students?
So training with ```python3 tools/train.py -c configs/det/ch_PP-OCRv4/ch_PP-OCRv4_det_teacher.yml``` should fine-tune the from you provided server model?
Do I have to use the pre-trained weights [LINK]([训练模型](https://paddleocr.bj.bcebos.com/PP-OCRv4/chinese/ch_PP-OCRv4_det_server_train.tar)) anyhow? I see in the config, that there is a different model being used as ```pretrained_model```.

Thanks!

**Comment by FocueAI** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/49752023/200720c9-1db8-4903-9d52-96ac53792596)

**Comment by UserWangZz** [comment]
pretrained_model: **https://paddleocr.bj.bcebos.com/pretrained/PPHGNet_small_ocr_det.pdparams**

In order to finetune the ppocrv4 server model, you can download the pretrained model from this link: https://paddleocr.bj.bcebos.com/PP-OCRv4/chinese/ch_PP-OCRv4_det_server_train.tar
then modify the pretrained_model to the path of the model.
